bin/filter/filter_service_std.py

```diff
@@ -43,19 +43,25 @@ def main():
         den = int(row['total_volume']) if row['total_volume'] else 0
 
         # performance = volume_meeting_target / total_volume
-        if num <= 0 or den <= 0:
+        if den <= 0:  # NaN calculation
             row['performance'] = None
         else:
             row['performance'] = max(round(num / den, 4), 0)
 
         # calculate target_met
-        if row['target']:
+        if row['target'] is not None:  # target can be 0
             target = float(row['target'])
 
-            # if no performance(volume_meeting_target/total_volume)
-            # then target_met is not possible
-            if row['performance'] is None:
-                row['target_met'] = 'NA'
+            # NaN or 0
+            if not row['performance']:
+                if row['volume_meeting_target'] is not None and row['total_volume'] is not None:
+                    # volume_meeting_target and total_volume are defined,
+                    # either can be 0, target_met is not met
+                    row['target_met'] = 'N'
+                else:
+                    # volume_meeting_target or total_volume are not defined,
+                    # target_met is not possible
+                    row['target_met'] = 'NA'
```

Comment:
Should be able to simplify this:
```suggestion
            if row['volume_meeting_target'] is None or row['total_volume'] is None:
                # volume_meeting_target or total_volume are not defined,
                # target_met is not possible
                row['target_met'] = 'NA'
```